scheduleit/views/meetings/edit.twig

```diff
@@ -25,7 +25,10 @@
       </div>
       <div class="modal-footer">
         <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
-        <button type="button" class="btn btn-danger">Delete</button>
+        <form method="post">
+          <input class="form-contrl" type="hidden" name="deleteHash" value="{{meeting.hash}}">
```

Comment:
Since this is a hidden input, it doesn't need any CSS. You can remove ` class="form-contrl"`.